src/shared/lib/GenericAssayUtils/GenericAssayCommonUtils.spec.ts

```diff
@@ -88,6 +88,61 @@ describe('GenericAssayCommonUtils', () => {
             );
         });
 
+        it('Shows entity_stable_id and description when name is missing in properties', () => {
```

Comment:
these tests seem to assert something different from what the name of the PR implies.  just curious how it works.

Reply:
@alisman Maybe rename title to `Generic Assay options should not show NA when name or description missing from genericEntityMetaProperties`

Reply:
This test is for `makeGenericAssayOption` function. These three tests will assert if `NAME` and `DESCRIPTION` property missing from `genericEntityMetaProperties`, will the label still correct.